packages/grid/frontend/frontend.dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:16-alpine as init-stage
+
+ENV ENVIRONMENT=${FRONTEND_ENV}
+
+WORKDIR /app
+
+COPY package.json yarn.lock .
+RUN yarn --frozen-lockfile
+
+COPY . .
+
+FROM node:16-alpine as grid-ui-development
+
+ENV NEXT_TELEMETRY_DISABLED=1
+
+WORKDIR /app
+COPY --from=init-stage /app .
+CMD ["/usr/local/bin/node", "--max_old_space_size=4096", "/app/node_modules/.bin/next", "dev", "-p", "80"]
```

Comment:
Seems like this is now kebab case: https://nodejs.org/api/cli.html#cli_max_old_space_size_size_in_megabytes